docs/content/users/usage/commands.md

```diff
@@ -667,11 +667,11 @@ ddev hostname somesite.ddev.local 127.0.0.1
 
 Flags:
 
-* `--extract-path`: If provided asset is an archive, provide the path to extract within the archive.
-* `--no-drop`: Set if you do NOT want to drop the db before importing.
-* `--progress`, `-p`: Display a progress bar during import. (default `true`)
-* `--src`, `-f`: Provide the path to a SQL dump in `.sql`, `.tar`, `.tar.gz`, `.tgz`, `.bz2`, `.xx`, or `.zip` format.
-* `--target-db`, `-d`: If provided, target-db is alternate database to import into. (default `"db"`)
+* `--database`, `-d`: Target database to import into (default "db")
+* `--extract-path`: Path to extract within the archive
+* `--file`, `-f`: Path to a SQL dump file, plain text or compressed (tar/tar.gz/tgz/zip)
+* `--no-drop`: Do not drop the database before importing
+* `--no-progress`: Do not output progress
```

Comment:
For stylistic consistency, literal values and file extensions should be wrapped in backticks—so I’d keep those. The new “plain text or compressed (...)” also seems less specific than what it replaced.

```suggestion
* `--database`, `-d`: Target database to import into (default `"db"`)
* `--extract-path`: Path to extract within the archive
* `--file`, `-f`: Provide the path to a SQL dump in `.sql`, `.tar`, `.tar.gz`, `.tgz`, `.bz2`, `.xx`, or `.zip` format.
* `--no-drop`: Do not drop the database before importing
* `--no-progress`: Do not output progress
```

Reply:
Done, also fixed in the command